concent_api/core/tests/utils.py

```diff
@@ -64,6 +64,18 @@ def _get_encoded_requestor_ethereum_different_public_key(self):
         """ Returns requestor ethereum public key encoded. """
         return '0x' + self._get_encoded_key(self.DIFFERENT_REQUESTOR_PUBLIC_KEY)
 
+    def _sign_message(self, golem_message, client_private_key = None, client_public_key = None):
```

Comment:
there is `sign_message` function in `utils/helpers.py`, it could (most probably) be reused here